addons/edi_stock/models/edi_move_cancellation_report_document.py

```diff
@@ -7,6 +7,18 @@
 _logger = logging.getLogger(__name__)
 
 
+class IrModuleModule(models.Model):
+    _inherit = "ir.module.module"
+
+    def _is_sale_stock_installed(self):
+        """
+        Returns bool() sale_stock module existing and being in state 'installed'.
+        With more time this be separated out into a bridge module
+        """
+        sale_stock_module = self.search([("name", "=", "sale_stock")])
+        return bool(sale_stock_module and sale_stock_module.state == "installed")
```

Comment:
(Moot if we decide we should be adding a EDI Sale/Stock linking module)

Nit: You could just include the state in the `sale_stock_module` domain. Probably a tiny bit more efficient then doing a search and then accessing the value from the module record.

Reply:
I think we should create the module.

Reply:
Yep can see that decision has now been made, will resolve. :)